frontend/packages/local-storage-operator-plugin/OWNERS

```diff
@@ -0,0 +1,15 @@
+reviewers:
+  - a2batic
+  - afreen23
+  - bipuladh
+  - cloudbehl
+  - gnehapk
+  - rawagner
+approvers:
+  - afreen23
+  - bipuladh
+  - cloudbehl
+  - gnehapk
+  - rawagner
```

Comment:
No need to add him. He's the OWNER of the whole console.

Reply:
Since, he was added in ceph and nooba plugin, I followed the same.

Reply:
Back then he was not.